### PR TITLE
Add blog post about Presto First Steps

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

File: _posts/2020-10-07-presto-first-steps.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+[Presto First Steps](https://learning.oreilly.com/live-training/courses/presto-first-steps/0636920462859/)
+in November.
+
+<!--more-->
+
+The new course is aimed at beginners with Presto, who want to accelerate their
+initial understanding and adoption. You ramp up quickly to install and configure
+Presto, use the CLI, and learn how to query connected data sources with SQL.
+
+The class includes three interactive training exercises on Katacoda. They allow
```
Comment (bitsondatadev) [inline _posts/2020-10-07-presto-first-steps.md:26]: A link to katacoda.com here might be useful for those who don't know about it.

File: _posts/2020-10-07-presto-first-steps.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+you to get hands on experience with Presto immediately. Lots of useful tips and
+tricks are covered in my material, and of course I plan to run a bunch of
+additional demos. You can find more details about the content of the class in
+[the registration page](https://learning.oreilly.com/live-training/courses/presto-first-steps/0636920462859/).
+
+The class is completely interactive, and I look forward to many of you joining me
```
Comment (bitsondatadev) [inline _posts/2020-10-07-presto-first-steps.md:32]: Perhaps you could move "The class is completely interactive up to the paragraph "The new interactive course...".

File: _posts/2020-10-07-presto-first-steps.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+and bring lots of great questions to ask. 
+
+So don't miss out and make sure you [reserve your ticket
```
Comment (bitsondatadev) [inline _posts/2020-10-07-presto-first-steps.md:35]: Remove So

**Review by bitsondatadev — COMMENTED** [review]
Looks good just a few suggestions.

File: _posts/2020-10-07-presto-first-steps.md (hunk 1 above)
Comment (mosabua) [reply _posts/2020-10-07-presto-first-steps.md:26]: done

File: _posts/2020-10-07-presto-first-steps.md (hunk 2 above)
Comment (mosabua) [reply _posts/2020-10-07-presto-first-steps.md:32]: Done

**Pull request merged**